cmd/lncli/cmd_payments.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/lightningnetwork/lnd/chainreg"
 	"github.com/lightningnetwork/lnd/lnrpc"
 	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	routerrpc2 "github.com/lightningnetwork/lnd/lnrpc/routerrpc/routerrpc"
```

Comment:
Wouldn't it be much cleaner if the proto files would be oved under a `proto` directory for each of these rpc subservers? Then the generated code and the rest of the subserver implementations can stay where they are.

Reply:
It was a choice I made based on the fact that anyone who has already imported them doesn't have to change them, and the size of the change set was smaller, for internal importers. I considered it the other way around but the changeset was way way bigger.

Reply:
Naming wise it's weird the way it's implemented in the current pr. Usually generated code either resides in a `pb` folder or the proto files in a `proto` folder. Perhaps. Keep in mind that Go users will need to change all references too.

Reply:
I favor `pb`. I didn't make it that way because of the ambiguities when multiple packages are pulled in, so keeping the names related was what I chose. The current state has several conflicting conventions, which I avered to the most dominant. 

Some guidance on precisely what I should do so I can finish this and get it merged and the utility of light importing of the generated protobuf/grpc/gw stuff would be great.

The way it is in the current PR precisely is focused on minimising name changes for importing Go apps. Some kind of compromise has to be made. Keep in mind that the more indistinct the names the more necessity there will be for import aliases. Overall, clarity of intent should be given by names and structure. One of the things I enjoy about working with LN stuff, there is less visible influence of either java or c++ programming conventions.